js/matchkeys/goldrush/goldrush.mjs

```diff
@@ -14,7 +14,7 @@ function loadMarcJson(record) {
     throw new Error('MARC fields[0] key is not numeric.');
   }
   if (!marcObj.leader) {
-    throw new Error('MARC leader field is missing.');
+    marcObj.leader = '00000nam a22000000a 4500';
```

Comment:
The reason I don't like doing it here is that it won't even log a warning about a missing leader.

Reply:
We can't just throw exceptions and ignore them and  log them.. If you initialize matchkeys for 80 million records the mergekey JS starts to fail we can't ignore 80 million thrown exceptions and log them. It must fail.

If JS has a Type system for exceptions we could start to use that and ignore some.

Another way would  be to extend the method we use for getting matchkeys so that it can produce warnings.

But even so if there are million of warnings.. is logging really a good idea? I don't think so.

Either the system accepts leaders or it doesn't. For the two OAI servers (Penn, NY) all records returned are not valid MARCXML .. The leaders (when present) are also not according to schema.

Reply:
I will approve this because it fixes the problem at hand. 

More generally, we need a way to skip broken records and log them out but this is a new piece of functionality that needs to specced out, not a quick fix. I'lll put in a ticket for it.